tests/conftest.py

```diff
@@ -43,3 +43,18 @@ def flat_settings(fs: FakeFilesystem) -> SettingsExample:
 
     # Create and return a Settings object for the test
     return SettingsExample("test_app", flat_config=True)
+
+
+@pytest.fixture()
+def xdg_settings(fs: FakeFilesystem) -> SettingsExample:
```

Comment:
can you add another test please, say `xdg_settings_from_env`, that changes the `XDG_CONFIG_HOME` env variable to something different so we can ensure that the env is being picked up?

if you see from the coverage, line 48 (that code path) is not being checked. 

It also looks like the function `_paths_from_env` is not actually used?

```
---------- coverage: platform linux, python 3.11.9-final-0 -----------
Name                                 Stmts   Miss  Cover   Missing
------------------------------------------------------------------
simple_toml_settings/__init__.py         2      0   100%
simple_toml_settings/exceptions.py       8      0   100%
simple_toml_settings/settings.py        70      0   100%
simple_toml_settings/xdg_config.py      14      6    57%   48, 74-80
------------------------------------------------------------------
TOTAL                                   94      6    94%
```

Reply:
you can set an Env variable in pytest for a specfic test by including the `monkeypatch` fixture in tthat test then using the [monkeypatch.setenv](https://docs.pytest.org/en/stable/reference/reference.html#pytest.MonkeyPatch.setenv) command

Reply:
a'ighty.  Well I do need to learn to write better python tests.

Reply:
> a'ighty. Well I do need to learn to write better python tests.

I'm crap at them myself, and they are generally the last thing I do! A good help is to look at the coverage report which will show you any lines that are not covered by tests.

Reply:
done.